packages/web3-utils/test/fixtures/validation.ts

```diff
@@ -28,41 +28,43 @@ export const compareBlockNumbersValidData: [[Numbers, Numbers], number][] = [
 	[[1, BigInt(1)], 0],
 	[[1, BigInt(2)], -1],
 	[[2, BigInt(1)], 1],
-
 	[['earliest', 0], 0],
+	[[0, 'earliest'], 0],
+	[['earliest', 'earliest'], 0],
 	[['pending', 'pending'], 0],
 	[['latest', 'latest'], 0],
```

Comment:
Bring back some deleted tests as they will be valid again if the suggestion to code was applied.
```suggestion
	[['latest', 'latest'], 0],
	[['earliest', 2], -1],
	[[2, 'earliest'], 1],
```

Reply:
added case for earliest.